Antreas dev

AntreasAntoniou opened the pull request.

justinengelmann commented on configs/learner/linear_layer_fine_tuning_learner.yaml:
```diff
@@ -3,6 +3,7 @@ defaults:
   - lr_scheduler_config: cosine_annealing.yaml
 
 _target_: gate.learners.single_layer_fine_tuning.LinearLayerFineTuningScheme
+name: linear_tune
```
Is there a neat way to take the .yaml file name for this through hydra?
I'm thinking that that would avoid A) having to type the same/similar name multiple times when creating a new config, and B) prevent accidentally mismatching this name and the .yaml name.

AntreasAntoniou replied:
The issue is that sometimes the identifier you want to use will be shorter than the full thing, all for the same of keeping the directory name short

justinengelmann reviewed (approved):
LGTM

AntreasAntoniou committed:
Updates

AntreasAntoniou committed:
Updates

AntreasAntoniou committed:
updates

AntreasAntoniou committed:
Segmentation task specific clip, timm and tali, and tests

AntreasAntoniou committed:
updates

AntreasAntoniou committed:
modify backbones to return num patches for image models

AntreasAntoniou committed:
pass segmentation tests, debugged code

AntreasAntoniou committed:
update

AntreasAntoniou committed:
Add multi class classification trainer and evaluator for medical experiments

AntreasAntoniou committed:
Updates

AntreasAntoniou committed:
Add HAM10K dataset

AntreasAntoniou committed:
add download check condition

AntreasAntoniou committed:
debug

AntreasAntoniou committed:
Debug

AntreasAntoniou committed:
updates

AntreasAntoniou committed:
Debug

AntreasAntoniou committed:
Debug

AntreasAntoniou committed:
debug

AntreasAntoniou committed:
debug

AntreasAntoniou committed:
feat: Add method to compute metrics for BackboneWithLinear adapter

This commit adds a `compute_metrics` method that calculates loss and
top-1 and top-5 accuracies using `F.cross_entropy` and `accuracy_top_k`
metrics, respectively. This method can be used to evaluate model
performance during training.

AntreasAntoniou committed:
fix: Use 'k' instead of 'top_k' in accuracy_top_k function

The 'accuracy_top_k' function in classification.py was modified to use the 'k' parameter instead of 'top_k' parameter. This ensures consistency with other parts of the code and removes a potential source of confusion.

AntreasAntoniou committed:
debug

AntreasAntoniou committed:
debug

AntreasAntoniou committed:
debug

AntreasAntoniou committed:
debug

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
debug

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
Updates

AntreasAntoniou committed:
updates

AntreasAntoniou committed:
update

AntreasAntoniou committed:
updates

AntreasAntoniou committed:
add explicit randaug transform

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update gate version

AntreasAntoniou committed:
Update

AntreasAntoniou committed:
Updates

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
fix env variable typo

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
add useful dev tools to dockerfile

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
update

AntreasAntoniou committed:
meeting notes

AntreasAntoniou committed:
fix conflicts

AntreasAntoniou committed:
Tell git-crypt to encrypt secrets

AntreasAntoniou committed:
Tell git-crypt to encrypt secrets

AntreasAntoniou committed:
Add video dataset tests

AntreasAntoniou committed:
Add tali updates

github-actions[bot] committed:
Apply black, isort, and autoflake formatting

AntreasAntoniou committed:
Apply formatting

AntreasAntoniou committed:
Apply formatting
